torch_geometric/data/download.py

```diff
@@ -2,11 +2,13 @@
 import ssl
 import sys
 import urllib
+from typing import Optional
 
 from .makedirs import makedirs
 
 
-def download_url(url: str, folder: str, log: bool = True):
+def download_url(url: str, folder: str, log: bool = True,
```

Comment:
Needed so that we can keep the downloaded files. Other approaches to downloading multiple files just delete or rename the raw data (for example https://github.com/pyg-team/pytorch_geometric/blob/9a16a0eb841e1efaedc06efb5b6b5bcf407bf588/torch_geometric/datasets/ged_dataset.py#L119). 

Of course both ways are fine.